share/spice/leak_db/leak_db.js

```diff
@@ -23,8 +24,13 @@
                 sourceUrl: 'http://leakdb.abusix.com/?q='+ encodeURIComponent(query),
                 sourceName: 'Abusix'
             },
+            normalize: function(item) {
+                return {
+                    title: 'hashes for "' + api_result.query + '"'
```

Comment:
I'm not sure if I like this title, but couldn't think of a better one. Any idea?